test/test_torch.py

```diff
@@ -2561,6 +2561,71 @@ def test(M):
     def test_det(self):
         self._test_det(self, lambda x: x)
 
+    @staticmethod
+    def _test_stft(self, conv_fn):
```

Comment:
I would be more confident in these tests if you added a few known sequences and check their stft. Things like constants, +1/-1/+1/-1/..., and maybe a few other simple signals.